.changeset/fresh-radios-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-kubernetes': minor
+'@backstage/plugin-kubernetes-backend': minor
+'@backstage/plugin-kubernetes-common': minor
+---
+
+Add pod metrics lookup and display in pod table
```

Comment:
So just to nit a little bit more, I think it would be nice to provide some instructions here of what the breaking changes are and how to update, given that we're doing minor bumps. Doesn't need to be a full on migration guide but just a bit of guidance of what APIs have changed.

Reply:
done